microsoft-edge/webview2/release-notes.md

```diff
@@ -46,7 +189,7 @@ For full API compatibility, this version of the WebView2 SDK requires WebView2 R
 
 ### General
 
-This WebView2 SDK release has the same bug fixes that are in WebView2 SDK 1.0.1466-prerelease. See **Bug fixes** in the following section.
+This WebView2 SDK release has the same bug fixes that are in WebView2 SDK 1.0.1414-prerelease. See **Bug fixes** in the following section.

```

Comment:
This WebView2 SDK release has the same [bug fixes](https://learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes?tabs=dotnetcsharp#101414-prerelease) that are in WebView2 SDK 1.0.1414-prerelease.